src/stability_sdk/client.py

```diff
@@ -428,7 +427,11 @@ def emit_request(
         help="engine to use for upscale",
         default="esrgan-v1-x2plus",
     )
-    
+
+    parser_animate = subparsers.add_parser('animate')
```

Comment:
very cool feature! should we also support command line interpolation for images in a specified input dir?

Reply:
ooh didn't think of that, yes that's a good idea! I have the UI behind --gui option and was thinking the other main way of using it would be passing in the project file and path to mp4 you would like created.